devops/docker/centos-8-amd64/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# TODO: Add MSFT mirror
+FROM centos:latest
+
+# CentOS Linux 8 had reached the End Of Life (EOL) on December 31st, 2021. 
+# It means that CentOS 8 will no longer receive development resources from
+# the official CentOS project. After Dec 31st, 2021, if you need to update 
+# your CentOS, you need to change the mirrors to vault.centos.org where they
+# will be archived permanently
```

Comment:
Until CentOS 8 will be removed from the list of officially targeted distros we need to have a package for it. This distro may have some serious user base.

Reply:
Agreed, i just placed the comment there to justify the changes from `mirror.centos.org` to `vault.centos.org`